monai/data/image_reader.py

```diff
@@ -244,7 +249,7 @@ def _get_affine(self, img) -> np.ndarray:
         affine = np.eye(direction.shape[0] + 1)
```

Comment:
Here if you declare `affine` to be type `np.ndarray` (ie. `affine: np.ndarray = np.eye(...)` which looks unnecessary) does this sort the mypy issue?